### PR TITLE
[ISSUE #1951] Datasource support Oracle

### PR DESCRIPTION
**Pull request by rom1c77**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/pom.xml [hunk 1]
```diff
@@ -246,6 +246,12 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
+            <artifactId>ojdbc8</artifactId>
+            <version>21.1.0.0</version>
```
Comment (yu199195) [inline shenyu-admin/pom.xml:253]: add runtime

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml [hunk 2]
```diff
@@ -32,7 +32,7 @@
     </resultMap>
 
     <sql id="Base_Column_List">
-        id, `type`, dict_code, dict_name, dict_value, `desc`, sort, enabled, date_created, date_updated
+        id, type, dict_code, dict_name, dict_value, `desc`, sort, enabled, date_created, date_updated
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:35]: why modify this？

File: shenyu-admin/src/main/resources/sql-script/oracle/schema.sql [hunk 3]
```diff
@@ -0,0 +1,2371 @@
+/* 删除表空间 */
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/sql-script/oracle/schema.sql:1]: add apache license and trsanfrom cn to en

File: shenyu-admin/src/main/resources/sql-script/oracle/schema.sql [hunk 4]
```diff
@@ -0,0 +1,2371 @@
...
+alter table SHENYU.USER_ROLE
+    add constraint PRIMARY_1 primary key (ID)
+    using index
+  tablespace USERS
+  pctfree 10
+  initrans 2
+  maxtrans 255;
+
+
+insert into shenyu.DASHBOARD_USER (id, user_name, password, role, enabled, date_created, date_updated)
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/sql-script/oracle/schema.sql:752]: used insert ingro into ?

File: shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml (hunk 2 above)
Comment (rom1c77) [reply shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml:35]: Type is not a keyword.so i remove `

File: shenyu-admin/pom.xml (hunk 1 above)
Comment (rom1c77) [reply shenyu-admin/pom.xml:253]: done

File: shenyu-admin/src/main/resources/sql-script/oracle/schema.sql (hunk 3 above)
Comment (rom1c77) [reply shenyu-admin/src/main/resources/sql-script/oracle/schema.sql]: done

File: shenyu-admin/pom.xml [hunk 5]
```diff
@@ -246,6 +246,13 @@
...
+            <scope>runtime</scope>
```
Comment (yu199195) [inline shenyu-admin/pom.xml:254]: remove version

File: shenyu-admin/src/main/resources/sql-script/oracle/schema.sql (hunk 4 above)
Comment (rom1c77) [reply shenyu-admin/src/main/resources/sql-script/oracle/schema.sql]: okok, I create producer about insert ingro..

File: shenyu-admin/src/main/resources/sql-script/oracle/schema.sql [hunk 6]
```diff
@@ -0,0 +1,2261 @@
...
+/* DROP TABLESPACE */
+DROP TABLESPACE shenyu_data INCLUDING CONTENTS AND DATAFILES CASCADE CONSTRAINTS;
+/* DROP USER  */
+DROP USER shenyu cascade;
+
+
+/* create tablespace   */
+create tablespace shenyu_data
+logging
+datafile 'D:\oracle\shenyu_data.dbf'
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/sql-script/oracle/schema.sql:27]: what is？

**Review by yu199195 — CHANGES_REQUESTED** [review]
1

**Review by yu199195 — CHANGES_REQUESTED** [review]
not mergerd

**Comment by yu199195** [comment]
hi， we already provide this fuction， thanks， i will close this pr

**Pull request closed without merging**